src/types/rtti.rs

```diff
@@ -1088,12 +1088,16 @@ impl Drop for Method {
 pub struct StaticMethod(red::CClassStaticFunction);
 
 impl StaticMethod {
-    pub fn new<R>(
+    pub fn new<C, R>(
         full_name: &CStr,
         short_name: &CStr,
         class: &Class,
         handler: FunctionHandler<IScriptable, R>,
-    ) -> PoolRef<Self> {
+        flags: FunctionFlags,
+    ) -> PoolRef<Self>
+    where
+        C: ScriptClass,
+    {
```

Comment:
```suggestion
    where
        C: ScriptClass,
    {
```
the C type is not used except for this bound, it should be removed

Reply:
done in 95ffdf4